config/pacbio_smrt_link_versions.yml

```diff
@@ -0,0 +1,159 @@
+---
+default: &default
+
+  yes_no: &yes_no ['Yes','No']
+  generate: &generate [In SMRT Link,Do Not Generate,On Instrument]
+  times: &times [10.0,15.0,20.0,24.0,30.0]
+
+  select_yes_no: &select_yes_no Yes,No
+  select_generate: &select_generate In SMRT Link,Do Not Generate,On Instrument
+  select_times: &select_times 10.0,15.0,20.0,24.0,30.0
+  
+  versions:
+    v10:
+      name: v10
+      active: true
+      default: true
+    v11:
+      name: v11
+      active: true
+      default: false
+
+  options:
+    ccs_analysis_output:
+      key: ccs_analysis_output
+      label: 'Ccs Analysis Output:'
+      default_value: 'Yes'
+      validations:
+        presence: {}
+        inclusion:
+          in: *yes_no
+      data_type: list
+      select_options: *select_yes_no
+      versions:
+        - v10
+    generate_hifi:
+      key: generate_hifi
+      label: 'Generate HiFi:'
```

Comment:
UI needs to append the colons to the labels of the input fields if we remove them. They are added to the traction DB after migration. It will be presented as the run's version's option. UI will be using them dynamically. I will check the traction-ui  to see how it is using them.

Reply:
Labels are with colons in the UI (in label-for). However, the same label text is used for "title" attribute without colons. I have removed the colons.